rs/nns/governance/canister/governance.did

```diff
@@ -541,7 +530,6 @@ type Proposal = record {
 type ProposalData = record {
   id : opt NeuronId;
   failure_reason : opt GovernanceError;
-  cf_participants : vec CfParticipant;
```

Comment:
So this seems like a breaking change for the clients who can't deserialize a vec into `null`. 

If we could do something like this, it would be fine, I think (if only we could hand craft these Candid files for NNS Gov).

See [here](https://github.com/dfinity/candid/blob/master/spec/Candid.md#primitive-types):

Subtyping 

```
-------------------------
<datatype> <: reserved
```

Coersion 

```
--------------------------
_ : <t> ~> null : reserved
```

Reply:
Are there clients that have that problem?

Reply:
There are no clients, apart from NNS Governance itself, that can read the `ProposalData` structure. Since NNS Governance itself reads that via stable memory (on upgrade), it seems safe to remove the field from the Candid API definition without any negative consequences.